docs/machine-learning/tutorials/phone-calls-anomaly-detection.md

```diff
@@ -0,0 +1,261 @@
+
+2. Create a directory named *Data* in your project to save your data set files.
+
+3. Install the **Microsoft.ML NuGet Package**:
+
+    [!INCLUDE [mlnet-current-nuget-version](../../../includes/mlnet-current-nuget-version.md)]
+
+    In Solution Explorer, right-click on your project and select **Manage NuGet Packages**. Choose "nuget.org" as the Package source, select the Browse tab, search for **Microsoft.ML** and select the **Install** button. Select the **OK** button on the **Preview Changes** dialog and then select the **I Accept** button on the **License Acceptance** dialog if you agree with the license terms for the packages listed. Repeat these steps for **Microsoft.ML.TimeSeries**.
+
+4. Add the following `using` statements at the top of your *Program.cs* file:
+
+    [!code-csharp[AddUsings](./snippets/phone-calls-anomaly-detection/csharp/Program.cs#AddUsings "Add necessary usings")]
+
+### Download your data
+
+1. Download the dataset and save it to the *Data* folder you previously created:
+
+   * Right click on [*phone-calls.csv*](https://raw.githubusercontent.com/dotnet/machinelearning-samples/master/samples/csharp/getting-started/AnomalyDetection_PhoneCalls/SrCnnDetection/Data/phone-calls.csv) and select "Save Link (or Target) As..."
+
+     Make sure you either save the \*.csv file to the *Data* folder, or after you save it elsewhere, move the \*.csv file to the *Data* folder.
+
+2. In Solution Explorer, right-click the \*.csv file and select **Properties**. Under **Advanced**, change the value of **Copy to Output Directory** to **Copy if newer**.
+
+The following table is a data preview from your \*.csv file:
+
+| timestamp  | value |
+|--------|--------------|
+| 2018/9/3  | 36.69670857  |
+| 2018/9/4  | 35.74160571  |
+| .....  | .....  |
+| 2018/10/3  | 34.49893429  |
+| ...    | ....   |
+
+### Create classes and define paths
+
+Next, define your input and prediction class data structures.
+
+Add a new class to your project:
+
+1. In **Solution Explorer**, right-click the project, and then select **Add > New Item**.
+
+2. In the **Add New Item dialog box**, select **Class** and change the **Name** field to *PhoneCallsData.cs*. Then, select the **Add** button.
+
+   The *PhoneCallsData.cs* file opens in the code editor.
+
+3. Add the following `using` statement to the top of *PhoneCallsData.cs*:
+
+   ```csharp
+   using Microsoft.ML.Data;
+   ```
+
+4. Remove the existing class definition and add the following code, which has two classes `PhoneCallsData` and `PhoneCallsPrediction`, to the *PhoneCallsData.cs* file:
+
+    [!code-csharp[DeclareTypes](./snippets/phone-calls-anomaly-detection/csharp/PhoneCallsData.cs#DeclareTypes "Declare data record types")]
+
+    `PhoneCallsData` specifies an input data class. The [LoadColumn](xref:Microsoft.ML.Data.LoadColumnAttribute.%23ctor%28System.Int32%29) attribute specifies which columns (by column index) in the dataset should be loaded.
```

Comment:
For this section, would it be possible to expand a little bit more about what the the input columns represent. The timestamp is kind of self explanatory but there's not a lot of information on the value column